extensions/amp-analytics/0.1/test/test-config.js

```diff
@@ -697,13 +739,17 @@ describes.realWin('AnalyticsConfig', {amp: false}, env => {
 
     const expandStringStub = sandbox.stub();
     expandStringStub.withArgs('CLIENT_ID(foo)').resolves('amp12345');
+    expandStringStub.withArgs('$NOT(foo)', {}).resolves('false');
```

Comment:
Can we change this slightly so that we are sure that we call `expandStringSync` with the result of `getMacros`?

I think this will work
```js
const macros = {};
expandStringStub.withArgs('$NOT(foo)', macros).resolves('false');
stubService(sandbox, win, 'amp-analytics-variables', 'getMacros').returns(
      macros);
```

Reply:
maybe also putting something in the macros object

Reply:
Good idea. Done